### PR TITLE
Fix typo: folp => foldp

### PR DESCRIPTION
**Pull request by renatorozas**

### CODE REVIEW

File: 030_elm_arch/actions.md [hunk 1]
```diff
@@ -196,7 +196,7 @@ main =
 
 <https://github.com/sporto/elm-tutorial-assets/blob/master/code/elm_arch/ActionsMultiple.elm>
 
-Every mouse click increases the count, every key press decreases the count. Note how we __merge__ the signals (mouseClickSignal and keyPressSignal) into one, as they both are of type `Signal.Signal Action` our folp can handle them.
+Every mouse click increases the count, every key press decreases the count. Note how we __merge__ the signals (mouseClickSignal and keyPressSignal) into one, as they both are of type `Signal.Signal Action` our foldp can handle them.
```
Comment (renatorozas) [inline 030_elm_arch/actions.md:199]: foldp, `foldp` or `Signal.foldp`?

Comment (renatorozas) [inline 030_elm_arch/actions.md:199]: I was actually tempted to change it to:

```
"...as they both are of type `Signal.Signal Action`, `Signal.foldp` can handle them."
```

But then I went with the minimal fix.

@sporto thoughts?

**Comment by sporto** [comment]
Thanks

**Comment by sporto** [comment]
I will leave this as just `foldp` as I think is clear where is it coming from at this point

**Pull request merged**